webapp/channels/src/components/forward_post_modal/forward_post_modal.tsx

```diff
@@ -77,7 +77,7 @@ const ForwardPostModal = ({onExited, post, actions}: Props) => {
 
     const canPostInSelectedChannel = useSelector(
         (state: GlobalState) => {
-            const channelId = isPrivateConversation ? channel.id : selectedChannelId;
+            const channelId = isPrivateConversation ? channel?.id : selectedChannelId;
```

Comment:
And here's another one where we have the ID from `post.channel_id`. Could you take a look through these to see where else we already have the channel ID?